lib/run.js

```diff
@@ -158,10 +158,12 @@ setups.firefox = function (browser, options, callback) {
         return 'user_pref("' + name + '", ' + prefs[name] + ');';
     }).join('\n');
 
-    options.options = options.options.concat([
-        '--no-remote',
-        '-profile', profileDir
-    ]);
+    if (options.profile === undefined || options.profile !== null) {
```

Comment:
I think this could just be `options.profile !== null`, no?

Reply:
Yeah my bad, I left it in when I was trying out separate options for with and without no-remote